src/read.rs

```diff
@@ -233,6 +234,46 @@ impl<R: Read + io::Seek> ZipArchive<R> {
         })
     }
 
+    /// Extract a Zip archive into a directory.
+    ///
+    /// Paths are sanitized so that they cannot escape the given directory.
```

Comment:
Can we add a note that this will bail on the first failure and will not clean up after itself if it encounters an error?